client/dashboard/profile-wizard/steps/store-details.js

```diff
@@ -54,6 +55,8 @@ class StoreDetails extends Component {
 		const { addNotice, goToNextStep, isError, updateSettings } = this.props;
 		const { addressLine1, addressLine2, city, countryState, postCode } = this.state;
 
+		recordEvent( 'storeprofiler_store_details', { store_country: countryState.split( ':' )[ 0 ] } );
```

Comment:
Out of curiosity, do we want to pass an object with a `store_country` param like this (as opposed to just recording an event `storeprofiler_store_country` with a string value of `US`) for the purpose of extending this param in the future?

Reply:
The second parameter here (event properties) is supposed to be an object, and tracks itself expects a property name and not just a string.

Reply:
Other properties also get appended to all event before storage in tracks (device, blog ID, etc).